latex/Specific Aims.tex

```diff
@@ -124,21 +124,25 @@
 
-{\bf \underline{Aim 1.} Generate new data for ``simple'' SAMPL blind challenges on physical property prediction}\\
+%{\bf \underline{Aim 1.} Generate new data for ``simple'' SAMPL blind challenges on physical property prediction}\\
+{\bf \underline{Aim 1.} Spur improvements in modeling protonation states, tautomers, and solvation via generation of new physical property datasets.}\\
 We will develop new solution-phase datasets for druglike small molecules. 
 These data can test critical aspects of small molecule modeling (including accounting for interactions and treatment of protonation/tautomeric state) and improve our ability to predict physical properties relevant to drug discovery in new regions of chemical space. 
 We will initially focus on distribution between organic phases and on pKa's and their modulation by solvent environment, using these data to drive improvements in the modeling of ligand interactions.
 
 
-{\bf \underline{Aim 2.} Measure binding of novel host-guest complexes for introductory ligand binding challenges.}\\
+%{\bf \underline{Aim 2.} Measure binding of novel host-guest complexes for introductory ligand binding challenges.}\\
+{\bf \underline{Aim 2.} Measure high affinity binding of drug-like compounds in simple receptors to challenge quantitative models of binding in systems lacking sampling challenges often observed in protein-ligand binding.}\\
 We will measure new host-guest binding free energies for cucurbiturils and deep-cavity cavitands, yielding further host-guest binding challenges which span between physical property prediction and protein-ligand binding. 
 Host guest systems are some of the simplest cases of molecular recognition, and thus these binding data will drive improvements in modeling of simple binding systems with techniques of relevance to drug discovery.
 
-{\bf \underline{Aim 3.} Generate biologically relevant advanced model systems for protein-ligand binding challenges.}\\
+%{\bf \underline{Aim 3.} Generate biologically relevant advanced model systems for protein-ligand binding challenges.}\\
+{\bf \underline{Aim 3.} Develop model protein-ligand systems that isolate specific modeling challenges of drug targets.}\\
 We will identify suitable biological protein-ligand model systems (difficult but tractable in order to push the limits of physical techniques) then measure binding and develop these for blind challenges. 
 This will include binding studies on human serum albumin and either bromodomains or aspartyl proteases; initial binding data will be expanded by the selection of additional ligands or the creation of mutations in the protein that modulate binding.
 
-{\bf \underline{Aim 4.} Coordinate, run, and analyze blind challenges to advance modeling of binding.} \\
+%{\bf \underline{Aim 4.} Coordinate, run, and analyze blind challenges to advance modeling of binding.} \\
+{\bf \underline{Aim 4.} Field iterated community blind challenges to advance quantitative biomolecular design.} \\
```

Comment:
That addition would require cutting a line somewhere else in the aims. :) If you think the additional "and related activities" is worth a line somewhere else, I'm open to suggestions. Otherwise I'd prefer to go with the concise version.